quinn-h3/src/qpack/tests.rs

```diff
@@ -82,7 +82,7 @@ fn blocked_header() {
         &mut enc_table.encoder(42),
         &mut block_buf,
         &mut enc_buf,
-        &[HeaderField::new("foo", "bar")],
+        vec![HeaderField::new("foo", "bar")].into_iter(),
```

Comment:
We should make sure that you can still pass an array or slice in.

Reply:
if we want `encode()` to accept an `IntoIterator<HeaderField>`, aren't we obligated to pass an owned value ?

Reply:
`IntoIterator` is implemented by the std library for slices.

Reply:
Fixed in d47c107